addons/test_discuss_full/tests/test_performance.py

```diff
@@ -1024,6 +1039,7 @@ def _get_init_messaging_result(self):
                 'im_status': 'bot',
                 'is_company': False,
                 'name': 'OdooBot',
+                'notification_preference': False,
```

Comment:
These deepEqual tests sucks. There's no reason to show `False` here for odoobot. A regression test would be best.

_(don't change anything in this PR, this is just a remark to show how poor this test is)_